tutorials/finetune_lora.md

```diff
@@ -164,18 +164,30 @@ python scripts/merge_lora.py \
   --out_dir "out/lora_merged/stablelm-base-alpha-3b/"
 ```
 
-After merging, we can use the `base.py` file for inference using the new checkpoint file. Note that if your new checkpoint directory is different from the original checkpoint directory, we also have to copy over the `*.json` files which are required for the configuration and tokenizer information:
+After merging, we can use the `base.py` file for inference using the new checkpoint file. Note that if your new checkpoint directory is different from the original checkpoint directory, we also have to copy over the tokenizer and config files:
 
 ```bash
-cp checkpoints/stabilityai/stablelm-base-alpha-3b/*.json \
+cp checkpoints/stabilityai/stablelm-base-alpha-3b/{tokenizer*,lit_config.json} \
```

Comment:
This only works in the fish shell btw. Most users will probably use bash or zsh and probably copy-paste this command only to run into an error.

Reply:
Perhaps you could list both commands separately just to be safe.

Reply:
This should be valid bash notation too, IDK about zsh though

Reply:
@awaelchli I am actually using zsh and it seems to work fine for me. If @carmocca  says that it works on bash for him, too, I think this would cover the most common use cases and this should be good to merge. 

Or do you think it's better to split it up into 2 separate commands?

Reply:
Note that all of the configs currently require a `tokenizer*` file present: https://github.com/Lightning-AI/lit-gpt/blob/main/lit_gpt/tokenizer.py#L15-L23

So the error would not happen under normal circumstances